docs/_includes/common/button.html

```diff
@@ -225,6 +225,38 @@ <h3 id="button-icon">Icon Button</h3>
 </button>
     {% endhighlight %}
 
+<h3>Button wrapping and overflow behavior</h3>
+
+<p>Text of buttons will wrap and overflow like their plain HTML companions.</p>
```

Comment:
Maybe add "when a fixed width is applied"? And also show the max width in the example code? I think it's good to remind people (including me!) that wrapping only kicks in when a fixed width is applied.

Reply:
Noted. Will add more to make the example clearer.

Reply:
Where I'm headed:

![image](https://user-images.githubusercontent.com/105656/46496412-37078000-c7d5-11e8-8b31-ea9f21b99ed6.png)

Reply:
Text only wraps when a fixed width is applied right? So we don't need the word "even" in the first example?

Reply:
Well, once it reaches 100% of any container, it will wrap, which is just like normal buttons:

![image](https://user-images.githubusercontent.com/105656/46497123-ebee6c80-c7d6-11e8-95b8-8872b4c1f5b2.png)

Reply:
Yeah, I guess we need to account for that in the wording too. How about "when the button fills it's constrained width (if any)". Or words to those affect.